gem/settings/base.py

```diff
@@ -216,7 +216,10 @@ def get_default_template(site_layout_base, site_layout):
     }
 
 
-SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'base')
+SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'chhaajja')
```

Comment:
can we please rename the templates to match the chhaa-jaa, there are too many different spellings of Chhaa Jaa